### ISSUE TITLE
Not working with crystal v1

### ISSUE THREAD

**Issue by Gembot-ai-admin** [amberframework/homebrew-amber#8]
Got this error when trying a fresh install of crystal and amber:

```bash
$ brew install crystal
==> Downloading https://homebrew.bintray.com/bottles/crystal-1.0.0.big_sur.bottle.tar.gz
Already downloaded: /Users/richardhp/Library/Caches/Homebrew/downloads/b66c1f68cc49f0ff49afaceb1f6af5cdc59526ead4d6baf8d21d4a3ef5823d41--crystal-1.0.0.big_sur.bottle.tar.gz
==> Pouring crystal-1.0.0.big_sur.bottle.tar.gz
==> Caveats
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d
==> Summary
🍺  /usr/local/Cellar/crystal/1.0.0: 1,194 files, 55.3MB
$ crystal --version
Crystal 1.0.0 (2021-03-22)

LLVM: 9.0.1
Default target: x86_64-apple-macosx
$ brew install amber
==> Installing amber from amberframework/amber
==> Downloading https://github.com/amberframework/amber/archive/v0.36.0.tar.gz
Already downloaded: /Users/richardhp/Library/Caches/Homebrew/downloads/acaa988da902da4ffb7ee377cc798a773b54f5f665f1a6f846447a8ae5955dbd--amber-0.36.0.tar.gz
==> shards install
Last 15 lines from /Users/richardhp/Library/Logs/Homebrew/amber/01.shards:
2021-04-08 13:46:14 +1200

shards
install

Resolving dependencies
Fetching https://github.com/amberframework/amber-router.git
Fetching https://github.com/crystal-lang/json_mapping.cr.git
Fetching https://github.com/crystal-ameba/ameba.git
Unable to satisfy the following requirements:

- `crystal (>= 0.35.0)` required by `amber_router 0.4.4`
- `crystal (~> 0.34, >= 0.34.0)` required by `json_mapping 0.1.0`
- `crystal (>= 0.35.0)` required by `ameba 0.13.4`
Failed to resolve dependencies, try updating incompatible shards or use --ignore-crystal-version as a workaround if no update is available.

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/amberframework/homebrew-amber/issues
```

**Comment by svenjr** [comment]
Has anyone solved this? Is there a way to build amber using crystal v1?

**Comment by elorest** [comment]
This is currently being worked on. Everything technically works but the issue is a lot of dependencies are locked to an older version of crystal. 

You can manually build by:

* Clone amber
* cd in
* Run `shards update --ignore-crystal-version`
* then compile from make or crystal build.

**Comment by crimson-knight** [comment]
Is there anything that you can throw out there for us in the community to pitch in and help get this working? I've tried downloading and building the source and running, but since I already had a version of amber installed through homebrew (and I can't run `brew remove amber` without getting an error that `amber` isn't a cask) I can't get the 1.0.0rc1 source to work after generating the new project.

I'm really interested in testing out Amber and using it for a start-up project I have beginning next month.

**Comment by elorest** [comment]
@crimson-knight Seth. In my tests rc1 will build without warnings and generate a project which will also build. If your seeing an error still please generate a pr with a fix or report the full issue and trace here.

You can probably remove with brew uninstall amberframework/amber

**Comment by crimson-knight** [comment]
@elorest I was able to clone from master, run `shards install` and then `make` and get the amber command to create a new application working and then successfully start up a new application. Of course I accomplished this right after posting how I was having an issue haha.

As far as removing the homebrew version, when I run `brew uninstall amber` OR `brew uninstall amberframework/amber` I get the same message:
`Error: No available formula or cask with the name "amber".`
`Error: No available formula or cask with the name "amberframework/amber".`

I definitely installed using homebrew, so I'm not sure what's going on here. I'll play around with it some more and see if I can figure it out.
